extensions-contrib/kubernetes-overlord-extensions/src/test/java/org/apache/druid/k8s/overlord/taskadapter/K8sTaskAdapterTest.java

```diff
@@ -572,14 +576,72 @@ void testEphemeralStorageIsRespected() throws IOException
     Assertions.assertEquals(expected, actual);
   }
 
+  @Test
+  void testEphemeralResourceIsEspected() throws IOException
```

Comment:
i think this test name is wrong, seems like it should be more like testCPUResourceIsRespected

Reply:
updated, thanks!